### ISSUE TITLE
Unknown update strategy: auto (RuntimeError)

### ISSUE THREAD

**Issue by pimjansen** [tinglesoftware/dependabot-azure-devops#128]
When running the script on default it throws below error:

Dependabot config file:
```yml
version: 2
updates:
  - package-ecosystem: "composer"
    directory: "/"
    schedule:
      interval: "weekly"
    ignore:
      - dependency-name: "*"
        update-types: ["version-update:semver-minor", "version-update:semver-major"]
```

Output
```
2021-05-28T13:59:08.8647591Z Using 'https' as protocol
2021-05-28T13:59:08.8648678Z Using 'dev.azure.com' as hostname
2021-05-28T13:59:08.8649259Z Using '443' as port
2021-05-28T13:59:08.8649751Z Using '' as virtual directory
2021-05-28T13:59:08.8650348Z Fetching composer dependency files for SenetDeApplicatiebouwers/Summa%20Digitaal%20Aanmelden%20LLO/_git/Aanmelden%20LLO
2021-05-28T13:59:08.8651149Z Targeting 'default' branch under '/' directory
2021-05-28T13:59:08.8651614Z Using 'auto' versioning strategy
2021-05-28T13:59:08.8651922Z Parsing dependencies information
2021-05-28T13:59:08.8652234Z Checking if doctrine/migrations 2.2.1 needs updating
2021-05-28T13:59:48.4599858Z /home/dependabot/dependabot-script/vendor/ruby/2.6.0/gems/dependabot-composer-0.143.6/lib/dependabot/composer/update_checker/requirements_updater.rb:53:in `check_update_strategy': Unknown update strategy: auto (RuntimeError)
```

The update strategy however is the default?
https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#versioning-strategy

**Comment by mburumaxwell** [comment]
@pimjansen,

1. Support for `update-types` is not yet live in this extension See #127 
2. `auto` should ideally not be used with `composer`. This is similar to npm/yarn and has been fixed in https://github.com/tinglesoftware/dependabot-azure-devops/commit/284be97e001fbdf6ddc458796211619b6f8c044d
3. Next release containing these changes, will be out in 2 days.

**Comment by pimjansen** [comment]
@mburumaxwell thanks for the reply. In the PR #127 i do not see anything about the update types? Furthermore what is the extension doing with the configurations? Isnt it just forwarding all where i use the native dependabot configuration file instead?

I expected it to just push all towards the dependabot core though

**Comment by mburumaxwell** [comment]
@pimjansen,

If you checked the changes in `update-script.rb` or read the latest version you would see the ignore conditions are now passed directly to dependabot. This is a new feature that wasn't there in the upstream dependabot until a few weeks ago, the script has to partly parse the values by itself. That particularly did not support the `update-types` field.

Further, the task execution file (`task.js`) had a bug that would not transfer the values from the configuration file to the docker container as described in #79 and fixed in #131.

**Comment by pimjansen** [comment]
Ok thanks for the nice reply!
